### PR TITLE
Update envoy-openssl and dependencies

### PR DESCRIPTION
**Pull request by tedjpoole**
Since the updated `envoy-openssl` now includes the `luajit2` fix, this PR includes reinstating the ppc and s390x architectures in the `maistra/scripts/update-deps.sh` script.
Also a minor addition to use bazel's `%workspace%` placeholder in `maistra/bazelrc-vendor`, rather than modifying it with `sed`.

### CODE REVIEW

**Review by dcillera — APPROVED** [review]
LGTM

File: maistra/scripts/update-deps.sh [hunk 1]
```diff
@@ -83,7 +83,7 @@ function copy_files() {
         cp_flags="-r"
       fi
       cp "${cp_flags}" "${f}" "${VENDOR_DIR}" || echo "Copy of ${f} failed. Ignoring..."
-      echo "build --override_repository=${repo_name}=/work/maistra/vendor/${repo_name}" >> "${BAZELRC}"
+      echo "build --override_repository=${repo_name}=%workspace%/maistra/vendor/${repo_name}" >> "${BAZELRC}"
```
Comment (jwendell) [inline maistra/scripts/update-deps.sh]: Any particular reason for this change? This will break downstream build, as it relies on /work.

**Review by jwendell — CHANGES_REQUESTED** [review]

File: maistra/scripts/update-deps.sh (hunk 1 above)
Comment (tedjpoole) [reply maistra/scripts/update-deps.sh]: It's a bit awkward for those who do not mount the workspace as `/work` (I mount mine to the same path in the container as on the host, which I find a lot easier). This change just made the scripts work regardless of where the workspace is mounted into the container

Comment (tedjpoole) [reply maistra/scripts/update-deps.sh]: Removed the use of `%workspace%` and reinstated `/work`

**Review by jwendell — APPROVED** [review]

**Pull request merged**